Fix for #4, #5, #6, #9

sbuser-git opened the pull request:
<!--
Thank you for your interest in and contributing to IBM Spectrum LSF RTM! There are a few simple things to check before submitting your pull request that can help with the review process. You should delete these items from your submission, but they are here to help bring them to your
attention.
-->
<!-- If this pull request resolves any bugs in the issues, provide a link: -->
Resolved #XXXX.
<!--
Have you followed the [contributor guidelines](https://github.com/IBM/ibm-spectrum-lsf-rtm-server/blob/main/CONTRIBUTING.md)?
Thank you for your contribution to IBM Spectrum LSF RTM!
-->

sbuser-git committed:
Update grid_benchmark_jobs.php

Signed-off-by: Stephanie Buser quic_sbuser@quicinc.com

Resolved #4, #5.

Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

sbuser-git committed:
Update grid_functions.php

Resolved #6.

Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

TheWitness commented on cacti/plugins/benchmark/grid_benchmark_jobs.php:
```diff
@@ -402,7 +403,7 @@ function stopRKey(evt) {
 							<option value='0'<?php if (get_request_var('clusterid') == '0') {?> selected<?php }?>><?php print __('All');?></option>
 							<?php
-							$clusters = grid_get_clusterlist();
+							$clusters = db_fetch_assoc('SELECT * FROM grid_clusters ORDER BY clustername');
```
This bit should be backed out.

sbuser-git replied:
Thanks, will back it out.

TheWitness reviewed (changes requested):
I believe that one line that I commented should be backed out, but the rest looks good.

sbuser-git committed:
Update grid_benchmark_jobs.php

Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

sbuser-git committed:
Update grid_manage_hosts.php

Issue #9

Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

ddb4github commented:
1. #5 diff is OK for me,  suggest to rename funcname `stopRKey` because it is not related to "Retrun" key now.
2. Why change `$job_exitinfo` to `-1`,  function `getExceptionStatus` has avoid to display `TERM_UNKNOWN`.
3. #4 diff: How about use Cacti core function `check_changed` in diff of function `process_request_var`
4. #9 diff is OK for me
5. #6 diff is OK for me

TheWitness commented:
@ddb4github, since the only thing that invalidates the "benchmark_id" is the "clusterid" change, I'm okay using that function as well.  It's your call.  You should be able to make the edits required to complete the merge right?

sbuser-git committed:
Update ss_grid_jobs.php

patch that disables the query cache for the Cacti Data Collectors - https://github.com/IBM/ibm-spectrum-lsf-rtm-server/issues/10

Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

sbuser-git committed:
Update ss_grid_slot_ut.php

patch that disables the query cache for the Cacti Data Collectors - https://github.com/IBM/ibm-spectrum-lsf-rtm-server/issues/10

Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

sbuser-git committed:
Update ss_grid_effectiveut.php

patch that disables the query cache for the Cacti Data Collectors - https://github.com/IBM/ibm-spectrum-lsf-rtm-server/issues/10

Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

sbuser-git committed:
Update ss_grid_lssched_bcs.php

patch that disables the query cache for the Cacti Data Collectors - https://github.com/IBM/ibm-spectrum-lsf-rtm-server/issues/10

Signed-off-by: Stephanie Buser <quic_sbuser@quicinc.com>

sbuser-git commented:
Add 4 commits from Larry for: patch that disables the query cache for the Cacti Data Collectors - https://github.com/IBM/ibm-spectrum-lsf-rtm-server/issues/10
